### ISSUE TITLE
Replace stock editor menu with a custom one

### ISSUE THREAD

**Issue by ussuri** [GoogleChrome/chromedeveditor#2258]
The editor menu uses the stock Chrome app menu. That makes its style quite different from the other 2 menus (especially after #1916) and leads to bogus menu items like "Look up in dictionary" on Mac (which doesn't even work, always showing "Result not found"). 

Try replacing with a custom menu with a revised and more useful set of actions. It would be especially nice to have it react to the file format and show actions appropriate just for that.
